esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -1751,6 +1751,19 @@ bool addWUQueryFilterTime(WUSortField *filters, unsigned short &count, MemoryBuf
     return true;
 }
 
+bool addWUQueryFilterTotalClusterTime(WUSortField& sortorder, WUSortField *filters, unsigned short &count, MemoryBuffer &filterBuf, unsigned milliseconds, WUSortField value)
+{
+    if (milliseconds == 0)
+        return false;
+
+    VStringBuffer vBuf("%d", milliseconds);
+    filters[count++] = value;
+    filterBuf.append(vBuf);
+
+    sortorder = WUSFtotalthortime;//Cassandra WUs can be filtered by TotalClusterTime only if sorting by TotalClusterTime.
```

Comment:
@wangkx - does this mean it will be unnecessarily sorting in the Dali case?

It may not be significant timing wise, but it would be good to avoid steps that are only applicable to one implementation.

Reply:
@jakesmith without this line,  the 'filtered by TotalClusterTime' will not work for CassandraWU. Should I change that logic in cassandrawu.cpp?

Reply:
@richardkchapman - can you check the logic here re. cassandra impl. needing to add a sort order in this case particularly?